common/remote/rmtsmtp.cpp

```diff
@@ -574,6 +586,8 @@ class CMailInfo
     {
         header.append(senderHeader).append(sender.get()).append("\r\n");
         header.append(toHeader).append(to.str()).append("\r\n");
+        if (!cc.isEmpty())
+            header.append(ccHeader).append(cc.str()).append("\r\n");
```

Comment:
Should bcc be here too?

Reply:
No.  This is within the DATA section, which is the visible portion of the email the recipients see.  It has nothing to do with delivering the email.  We omit the bcc recipients to satisfy the "blind" part of the bcc definition.

Reply:
Fair enough